### DISCUSSION TITLE
modify item based on its exe and not title

### DISCUSSION THREAD

**Discussion by pavichokche** [moudey/Shell#292] [Q&A] [answered]
I want to hide certain programs' action items in the context menu, but they don't all just have the program's name in their action item title, or it's a generic word. For example, Microsoft Access has a 'Preview' item that appears for some file types, and I never use that software. I just want to hide all items where "MSACCESS.EXE" is the command executable.

I tried things like this but nothing worked:
		item(where=this.exe = "msaccess.exe" vis="remove")

Unfortunately the documentation is so limited and lacking of examples to give context to how commands should/could be used, I'm quite lost...

**Comment by RubicBG** [comment, marked as answer]
this code is simple and it will work:

`modify(find='Open As Read-Only|Open in Protected View|"new"|Preview' where=(sel.file.ext=='.accdb' or sel.file.ext=='.maw') vis=remove)`

you can use NirSort ShellMenuView ( http://www.nirsoft.net/utils/shell_menu_view.html ) to see which extensions are associated with which commands.

PS: what you are actually asking for the moment is not achievable with simple commands. You can do complex logic: search the windows registry for the commands for the corresponding extension, check if the command contains the executable you are looking for ... 

I was able to create something similar with another command where I check 5 paths in the registry to get the same desired result

**Reply by pavichokche** [reply]
Thanks for the reply! Using shell menu view I was actually just able to disable all of the microsoft access associated items, not just see what their names/extensions are!
